config/config-example.js

```diff
@@ -210,6 +210,15 @@ exports.appealurl = '';
 exports.replsocketprefix = './logs/repl/';
 exports.replsocketmode = 0o600;
 
+// github - this is used for update alerts for a GitHub repository.
```

Comment:
Also, I think it should should specify that secret should be generated using `ruby -rsecurerandom -e 'puts SecureRandom.hex(20)'`.